testing/common.sh

```diff
@@ -604,7 +604,7 @@ function install_operator_version() {
   echo "Target operator release: $version"
 
   # Initialize the MinIO Kubernetes Operator
-  kubectl apply -k "${SCRIPT_DIR}/../resources"
+  kubectl apply -k github.com/minio/operator/resources/\?ref=v"$version"
```

Comment:
Why go to github again if we have all the code locally?

Reply:
Or do you want to test against another version that has already been released?

Reply:
Yes @ramondeklein if you follow the test in deploy-tenant-upgrade.sh will notice this test upgrades from the latest published operator to the code in the branch
https://github.com/minio/operator/blob/56cd580c08c5251a06a18e063b2131f2092babde/testing/deploy-tenant-upgrade.sh#L103-L111

Reply:
The local code is not from the required version. For this particular test, we need to assess it against a different version that has already been released. So, yes, that's the reason.